e2e/cypress/integration/system_console/environment_spec.js

```diff
@@ -284,6 +284,16 @@ describe('Environment', () => {
         });
     });
 
+    describe('System Console/Environment/Web Server', () => {
+        it('MM-T963 Configuration - Purge caches', () => {
+            cy.visit('/admin_console/environment/web_server');
+
+            cy.get('#PurgeButton').scrollIntoView().should('be.visible').within(() => {
+                cy.findByText('Purge All Caches').should('be.visible').click().wait(TIMEOUTS.ONE_SEC);
+            });
+        });
```

Comment:
PR ask was to confirm `Nothing crashes` lmk if there's already a standard way to assert on that! 😄

Reply:
@alejandrovelez7 Thanks for working on this e2e. There isn't really a standard way of asserting no crash was found, but you can probably just include additional step to reload the page and assert that the page loaded fine, like asserting on the admin consoler header `Web Server`

Reply:
Applied that suggestion in https://github.com/mattermost/mattermost-webapp/pull/9023/commits/93b3dda6a5e27c25cb281f07ab93d28c49c76a42 and spec is passing

![image](https://user-images.githubusercontent.com/26886607/135924878-76a3b261-126c-4d85-ab21-c6d3ca4cad0a.png)

Let me know if you have any additional feedback!